torch/csrc/jit/python/pybind_utils.h

```diff
@@ -626,6 +626,14 @@ inline std::string friendlyTypeName(py::handle obj) {
   }
 }
 
+// Thrown when trying to create a schema for a list of python
+// arguments that cannot be converted. 
+// Can be caught by the caller to attempt to use other schema
+// when there is an overloaded operator.
+struct schema_match_error : public std::runtime_error {
+  using std::runtime_error::runtime_error;
```

Comment:
What's this for?

Reply:
using is weird c++ syntax for reusing the constructors of runtime_error